src/analyses/apron/relationAnalysis.apron.ml

```diff
@@ -196,15 +196,14 @@ struct
   let assert_type_bounds ask rel x =
     assert (RD.Tracked.varinfo_tracked x);
     match Cilfacade.get_ikind x.vtype with
-    | ik when not (IntDomain.should_ignore_overflow ik) -> (* don't add type bounds for signed when assume_none *)
+    | ik -> (* don't add type bounds for signed when assume_none *)
```

Comment:
This comment is now wrong.

But also, what's the reason for removing this? Weren't the loop counters changed to `unsigned long long`?

Reply:
Thanks, I removed the comment. 

The reason for removing this was that the termination for some simple examples could not be established, when the loop counter in the original program was some `signed int` variable that was decremented. See this [example](https://github.com/goblint/analyzer/blob/d9f39105b35e4c65ac01b457cdb19ab82ac17c87/tests/regression/78-termination/50-decreasing-signed-int.c). 

With this change, we now succeed to show termination on 359 programs instead of 303 in sv-comp.